iele-examples/simpleOpenAuction.iele

```diff
@@ -70,7 +105,7 @@ define public @settle.auction() {
   %highest.bid = sload @highest.bid
   %beneficiary = sload @beneficiary
   %gas = call @iele.gas()
-  %status = call @deposit at %highest.bidder() send %highest.bid, gaslimit %gas
+  %status = call @deposit at %beneficiary() send %highest.bid, gaslimit %gas
```

Comment:
Shouldn't this function return the highest bidder so that the caller knows who to give the goods to?

Reply:
I made that fix as well. See the new comments.